src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1508,6 +1514,23 @@ namespace ReportSizingManager {
 						}
 					}
 					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirInletTempSizing ) {
+					if ( DesicDehum( DataDesicDehumNum ).RegenInletIsOutsideAirNode ) {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatOutTemp;
+					} else {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatRetTemp;
+					}
+					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirOutletTempSizing ) {
+					AutosizeDes = DesicDehum( DataDesicDehumNum ).RegenSetPointTemp;
+					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirInletHumRatSizing ) {
+					if ( DesicDehum( DataDesicDehumNum ).RegenInletIsOutsideAirNode ) {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatOutHumRat;
+					} else {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatRetHumRat;
+					}
+					bCheckForZero = false;
```

Comment:
Could these be combined with existing SizingType = HeatingWaterDesAirInletTempSizing ? For example using the DataDesicRegCoil flaghere? 
    // coil inlet temperature
    if ( CurOASysNum == 0 && PrimaryAirSystem( CurSysNum ).NumOAHeatCoils > 0 ) {
	AutosizeDes = OutAirFrac * FinalSysSizing( CurSysNum ).PreheatTemp + ( 1.0 - OutAirFrac ) * FinalSysSizing( CurSysNum ).HeatRetTemp;
    } else {
	AutosizeDes = OutAirFrac * FinalSysSizing( CurSysNum ).HeatOutTemp + ( 1.0 - OutAirFrac ) * FinalSysSizing( CurSysNum ).HeatRetTemp;
    }

If not I would rather see a more generic SizingType name so others could generically use this code. For example, instead of DesiccantRegCoilDesAirInletTempSizing, use HeatingCoilDesAirInletTempSizing. If this can be combined, the SizingType name should be changed to HeatingCoilDesAirInletTempSizing (there are only 6 occurrences in code)